bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -14,6 +14,7 @@
 
 namespace Pimcore\Bundle\CoreBundle\DependencyInjection;
 
+use Pimcore\Bundle\EcommerceFrameworkBundle\DependencyInjection\Config\Processor\PlaceholderProcessor;
```

Comment:
hmm, could you move the placeholderprocesser into corebundle and reference it in ecommerce bundle. 
feels wrong the other way round.

Reply:
Update done.

Reply:
- Placeholderprocessor moved.
- Documentation extended.